packages/integration-tests/index.js

```diff
@@ -102,34 +105,6 @@ async function getClusterArn(stackName) {
   return matchingArns[0];
 }
 
-/**
- * Get the template JSON from S3 for the workflow
- *
- * @param {string} stackName - Cloud formation stack name
- * @param {string} bucketName - S3 internal bucket name
- * @param {string} workflowName - workflow name
- * @returns {Promise.<Object>} template as a JSON object
- */
-function getWorkflowTemplate(stackName, bucketName, workflowName) {
```

Comment:
Out of curiosity, are the functions removed here also exported from this file? I ask because I ran into that on some other work and it creates confusing dependency chains

Reply:
They are, and they are still imported by this module for those compatibility reasons.

Reply:
I'm not delving into it just now, but I'm assuming there's a reason we can't just refactor it, given this is an integration test package and there shouldn't be external dependencies?